server/src/main/java/io/druid/server/QueryResource.java

```diff
@@ -95,11 +97,13 @@ public QueryResource(
   {
     this.config = config;
     this.jsonMapper = jsonMapper;
+    this.asciiSafeJsonMapper = jsonMapper.copy().configure(JsonGenerator.Feature.ESCAPE_NON_ASCII, true);
```

Comment:
There have been a number of fixes for copy() since the 2.4 branch. Notably https://github.com/FasterXML/jackson-databind/issues/696